src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./modal.css";
+function Modal({ id, name, description, timer, breakTimer, deleteTodo }) {
```

Comment:
Here.. The tasks are displayed as cards. So you can rename this as taskCard or something instead of Modal. As the modal function as a Popup onclick of a button.